tests/php/InlineCssTest.php

```
@@ -185,6 +185,108 @@ private function make_wp_styles( array $registered, array $queue, $path_value )
+
+	/**
+	 * Reset the once-per-process inline-drift logger flag for a drift test.
+	 *
+	 * The flag is held in a private static, so reflection is required to make
+	 * drift-triggering tests order-independent and observe the log path.
+	 */
+	private function reset_inline_drift_logger_flag(): void {
+		$prop = new \ReflectionProperty( Cache::class, 'inline_drift_logged' );
+		$prop->setAccessible( true );
+		$prop->setValue( null, false );
+	}
```

PHPMD (2.15.0)

[error] The method reset_inline_drift_logger_flag is not named in camelCase. (undefined)

(CamelCaseMethodName)